projects/lib-shared-modules/src/lib/components/search/search.component.ts

```diff
@@ -1,12 +1,20 @@
-import { Component } from '@angular/core';
+import { Component, EventEmitter, Output } from '@angular/core';
+import {MatButtonModule} from '@angular/material/button';
+import {MatFormFieldModule} from '@angular/material/form-field';
+import {MatIconModule} from '@angular/material/icon';
+import {MatInputModule} from '@angular/material/input';
 
 @Component({
   selector: 'lib-search',
   standalone: true,
-  imports: [],
+  imports: [MatButtonModule,MatFormFieldModule,MatIconModule,MatInputModule],
   templateUrl: './search.component.html',
-  styleUrl: './search.component.css'
+  styleUrl: './search.component.scss'
 })
 export class SearchComponent {
```

Comment:
Please add 
1. debouce timers.
2. minimum characters length property for search (Characters length should comes from parent and it's also should have default value if it's not passed from parent )